conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -38,18 +41,17 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-
+conda install --yes conda-build==1.18.2
```

Comment:
Looks like we might be able to drop it after all. See this PR ( https://github.com/conda-forge/conda-smithy/pull/152 ).